e2e/ruby/rails/demo/Gemfile

```diff
@@ -68,7 +68,6 @@ group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
   gem "selenium-webdriver"
-  gem "webdrivers"
 end
 
-gem "highlight_io", path: "../../../highlight"
+gem "highlight_io", path: "../../../../sdk/highlight-ruby/highlight"
```

Comment:
Maybe we should not change this path?

Reply:
this was from moving the e2e ruby app to the `e2e` directory; the sdk didn't move. does that make sense?

Reply:
Oh, I see - sorry!